packages/govern-server/src/api-graphql/schema.ts

```diff
@@ -45,7 +44,7 @@ export default gql`
     id: ID!
```

Comment:
To keep consistent with the naming we may want to rename everywhere `OptimisticQueue` for `GovernQueue`